src/javascript/app/base/binary_pages.js

```diff
@@ -65,86 +66,89 @@ const WhyUs              = require('../../static/pages/why_us');
 
 /* eslint-disable max-len */
 const pages_config = {
-    account_transfer         : { module: AccountTransfer,            is_authenticated: true, only_real: true, needs_currency: true },
     accounts                 : { module: Accounts,                   is_authenticated: true, needs_currency: true },
     api_tokenws              : { module: APIToken,                   is_authenticated: true },
+    authenticate             : { module: Authenticate,               is_authenticated: true, only_real: true },
     assessmentws             : { module: FinancialAssessment,        is_authenticated: true, only_real: true },
     asset_indexws            : { module: AssetIndexUI },
-    authenticate             : { module: Authenticate,               is_authenticated: true, only_real: true },
+    account_transfer         : { module: AccountTransfer,            is_authenticated: true, only_real: true, needs_currency: true },
     authorised_appsws        : { module: AuthorisedApps,             is_authenticated: true },
+    cfds                     : { module: GetStarted.CFDs },
+    careers                  : { module: StaticPages.Careers },
```

Comment:
`ca` is beofre `cf`, I'll just move this card back and you can re-check the alphabetical ordering of all these.